src/Components/Inquiry/Views/InquiryBasicInfo.tsx

```diff
@@ -21,27 +21,50 @@ import {
 import { useUpdateMyUserProfile } from "Components/Inquiry/Hooks/useUpdateMyUserProfile"
 import { logger } from "Components/Inquiry/util"
 import { Form, Formik } from "formik"
-import { ActionType, ContextModule, EditedUserProfile } from "@artsy/cohesion"
+import {
+  ActionType,
+  ContextModule,
+  EditedUserProfile,
+  EditProfileModalViewed,
+} from "@artsy/cohesion"
 import { useTracking } from "react-tracking"
 import { useAnalyticsContext } from "System/Hooks/useAnalyticsContext"
+import { useOnce } from "Utils/Hooks/useOnce"
 
 interface InquiryBasicInfoProps {
   artwork: InquiryBasicInfo_artwork$data
   me: InquiryBasicInfo_me$data | null | undefined
 }
 
 const InquiryBasicInfo: React.FC<InquiryBasicInfoProps> = ({ artwork, me }) => {
-  const { next, setContext, relayEnvironment } = useInquiryContext()
+  const { next, setContext, relayEnvironment, context } = useInquiryContext()
 
   const { submitUpdateMyUserProfile } = useUpdateMyUserProfile({
     relayEnvironment: relayEnvironment.current as Environment,
   })
 
   const { sendToast } = useToasts()
-
   const { trackEvent } = useTracking()
+
   const { contextPageOwnerType } = useAnalyticsContext()
 
+  useOnce(async () => {
+    await submitUpdateMyUserProfile({
+      promptedForUpdate: true,
+    })
```

Comment:
@araujobarret just FYI that it was more straightforward to update this timestamp when the modal is displayed, rather than updating it for each action that can be performed on the modal.

Reply:
You don't need to await this since the response doesn't matter

Reply:
Question, as far as I know only when touching the button `I haven't started` should update it, not when closing the modal. Can you double-check with product please?